lib/client.js

```diff
@@ -378,6 +379,19 @@ class AssetComputeClient extends EventEmitter {
         }
         return response;
     }
+
+    async checkEventJornal(journalUrl) {
```

Comment:
There is a typo in the name. Should be `checkEventJournal` (the `u` in journal is missing).